database/things/fp3.pldb

```diff
@@ -0,0 +1,22 @@
+title fp3
```

Comment:
this should be `fp`. dupes in the title field are fine. The only place we need a unique id is the filename.

Reply:
Got it!